test/src/main/java/org/apache/accumulo/harness/AccumuloClusterHarness.java

```diff
@@ -142,14 +142,20 @@ public void setupCluster() throws Exception {
         standaloneCluster.setAccumuloHome(conf.getAccumuloHome());
         standaloneCluster.setClientAccumuloConfDir(conf.getClientAccumuloConfDir());
         standaloneCluster.setHadoopConfDir(conf.getHadoopConfDir());
-        standaloneCluster.setServerCmdPrefix(conf.getServerCmdPrefix());
-        standaloneCluster.setClientCmdPrefix(conf.getClientCmdPrefix());
+        // These cannot be null
+        standaloneCluster
+            .setServerCmdPrefix(conf.getServerCmdPrefix() == null ? "" : conf.getServerCmdPrefix());
+        standaloneCluster
+            .setClientCmdPrefix(conf.getClientCmdPrefix() == null ? "" : conf.getClientCmdPrefix());
         cluster = standaloneCluster;
 
         // For SASL, we need to get the Hadoop configuration files as well otherwise UGI will log in
         // as SIMPLE instead of KERBEROS
-        Configuration hadoopConfiguration = standaloneCluster.getHadoopConfiguration();
+        // Note that getting the Hadoop config creates a servercontext which wacks up the
+        // AccumuloClientIT test so
+        // if SASL is enabled then the testclose() will fail
```

Comment:
Could format the comment a little better.  I also noticed some formatting to other classes when running "mvn clean verify -Dit.test=AccumuloClientIT -Dtest=foo -Dspotbugs.skip -Daccumulo.it.properties=/sw/uno/install/accumulo-2.0.0-SNAPSHOT/conf/accumulo-it.properties".

Reply:
@milleruntime The first error you have is because you have no users defined in your accumulo-it.properties file.  These are documented in the Testing.md file.  Can you please provide more insight into the class/comment formatting errors.  I ran the above and do not see anything identified but I probably do not know what to look for.  Is there a file produced with warnings?  Sorry.

Reply:
Ah you are correct!  Thanks.  I will update uno to include some dummy users.